src/json_decode.mli

```diff
@@ -208,11 +208,26 @@ and a value that is successfully decoded with the given decoder, [Error of strin
   Js.log \@\@ Js.Json.parseExn {| { "x": 23, "y": "b" } |} |> Decode.(field "y" int)
   (* prints [Error ...] *)
   Js.log \@\@ Js.Json.parseExn "123" |> Decode.(field "x" int)
-  (* prints [Ok None] *)
+  (* prints [Error ...] *)
```

Comment:
This was inconsistent with [this test](https://github.com/BuckleTypes/bs-json/pull/1/files#diff-16a01d1ae508be59dcab882fb423315cR246) previously implemented [here](https://github.com/BuckleTypes/bs-json/pull/1/files#diff-16a01d1ae508be59dcab882fb423315cL317).